eva/executor/create_index_executor.py

```diff
@@ -62,8 +59,6 @@ def exec(self):
 
         if IndexType.is_faiss_index_type(index_type):
             self._create_faiss_index()
-        else:
```

Comment:
Won't this result in saying the index is successfully created?

Reply:
Added `assert`